example_config.py

```diff
@@ -5,8 +5,14 @@
 Example userstorage configuration module.
 """
 
+import userstorage
 from userstorage import LoopDevice, Mount, File
 
+import pytest
+
+
+userstorage.missing_handler = pytest.xfail
```

Comment:
Since this is example code, we want to a comment here like:

    # Mark tests as xfail when the storage they need is missing.